docker/docker-compose.yml

```diff
@@ -58,6 +59,11 @@ services:
     container_name: converters_container
     ports:
```

Comment:
Remove `ports`. It will start to get very confusing on the server.

Reply:
I guess since this PR is not related to converters and looks like there is an unsolved issue related to docker of convertors we can handle this out of this PR.